### PR TITLE
#56

### PR DESCRIPTION
**Pull request by mtewes**
Weight training. Opening the PR, will not merge until I checked that it works.

### CODE REVIEW

**Comment by mtewes** [comment]
This seems to work, will merge in soon. @kuntzer want to have a look? Nothing really new, except the error function doing a proper normalized weighting.

**Comment by kuntzer** [comment]
Having a look now

File: tenbilac/err.py [hunk 1]
```diff
@@ -276,6 +305,29 @@ def msbwinvsquare(predictions, targets, auxinputs):
 
 
 
+def msbwsignorm(predictions, targets, auxinputs):
```
Comment (kuntzer) [inline tenbilac/err.py]: Why is this function appearing twice?

Comment (mtewes) [inline tenbilac/err.py]: Fixed, thanks!

**Pull request merged**